src/eventDispatcher.js

```diff
@@ -35,14 +35,15 @@ export class EventDispatcher {
      *  Triggers a given event.
      *  @param {String} eventName. 
      */
-    trigger(eventName) {
+    trigger(eventName, num) {
 
         Assert.isString(eventName);
+        Assert.isNumber(num);
```

Comment:
As I mentioned, we are going to trigger various types of data, sometimes we are just going to trigger an event without anything, so this Assert does not make sense.